pkg/v1/tkg/tkgpackageclient/image_utils.go

```diff
@@ -0,0 +1,48 @@
+// Copyright 2021 VMware, Inc. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package tkgpackageclient
+
+import (
+	"strings"
+
+	dockerParser "github.com/novln/docker-parser"
```

Comment:
Synced up with @blc1996 offline that we should look at removing introducing of this dependency and see if we can use imgpkg or alternate approach.

Reply:
Switched to use https://github.com/google/go-containerregistry